examples/sqlitetst/go.mod

```diff
@@ -0,0 +1,5 @@
+module hradek.net/go-sqlite
```

Comment:
Is this intentional? Shouldn't it be something generic like `sqlitetst`?

Reply:
That was an oversight from previous tests. will fix.

Reply:
As it is just a test module i changed it now to

module github.com/Skeeve/sqlitetst

Hope that's okay.

Reply:
Wouldn't go modules be confused by this repo containing a copy of the original `github.com/Skeeve/sqlitetst`? I would suggest to keep the name "local" (just `sqlitetst`) to avoid any kinds of collisions in the future.

Reply:
fixed.